.travis.yml

```diff
@@ -11,6 +11,7 @@ install:
   - go get golang.org/x/tools/cmd/goimports
   - ./hack/travis/install-etcd.sh
   - ./hack/build-go.sh
+  - export PATH=./third_party/etcd:$PATH
```

Comment:
Can we make travis also have $GOPATH/bin? I'm guessing that travis is doing that for us already, but being explicit is better.